resource/decker/support/plugins/handout/handout.js

```diff
@@ -0,0 +1,370 @@
+let Reveal;
+
+let centralSlide;
+
+let handoutSlideMode = false;
+const fakeRevealContainer = document.createElement("div");
+const fakeSlideContainer = document.createElement("div");
+fakeRevealContainer.appendChild(fakeSlideContainer);
+
+function undoAutomaticSlideAdjustments(slideElement) {
+  let slides = slideElement.querySelectorAll("section");
+  for (const slide of slides) {
+    slide.inert = false;
+    slide.hidden = false;
+    slide.removeAttribute("aria-hidden");
+  }
+}
+
+function findAllRealSlides(slidesElement, slideList) {
+  for (const child of slidesElement.childNodes) {
+    if (child.tagName !== "SECTION") continue;
+    if (child.classList.contains("stack")) {
+      for (const subchild of child.childNodes) {
+        if (subchild.tagName !== "SECTION") continue;
```

Comment:
getElementsByTagName("section")

Reply:
The whole codeflow of this part of the plugin has changed and should be using getElementsByTagName or proper querySelectors now.